appcelerator/titanium_mobile
# fix(android): specify default inspector context

###### NOTE: REQUIRES https://github.com/appcelerator/titanium_mobile/pull/11294

- Specify default inspector client context

##### TEST CASE
```
appc run -p android --debug-host localhost:8989
```
1. Open `chrome-devtools://` link in chrome-based browser.
2. Open Console, enter `Titanium`.
3. Should not experience crash.

##### DEMO
<img width="640px" src="https://jira.appcelerator.org/secure/attachment/67169/console.gif"/>

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-27527)
